client-mobile/src/components/molecules/ShoppingListButtons/styles.ts

```diff
@@ -0,0 +1,19 @@
+import { Sx } from "dripsy"
+
+export const shoppingListButtonsDefaultStyles: {
+  button: Sx
+  recipeButton: Sx
+  wrapper: Sx
+} = {
+  button: {
+    flexBasis: "60%",
+  },
+  recipeButton: {
+    flexBasis: "35%",
+  },
+  wrapper: {
+    flexBasis: "40",
```

Comment:
```suggestion
button: {
    flexBasis: "auto",
    flexGrow: 1,
  },
  recipeButton: {
    flexBasis: "35%",
  },
  wrapper: {
    flexBasis: 40,
    flexDirection: "row",
    gap: "$12",
```

This will ensure that there's always exactly 12 gap between buttons